components/AssetImage.vue

```diff
@@ -40,23 +42,22 @@ const isVerificationInvalid = computed(
   () => unref(props.image)?.verified === 'invalid'
 )
 
-const contentWidth = useElementSize(contentRef.value)
-
-const isLarge = computed(() => {
-  return contentWidth.width.value > 150
-})
+const isLarge = computed(() => contentWidth.value > LARGE_IMAGE_BREAKPOINT)
 
 watchEffect(() => {
   imageSrc.value = unref(props.image?.url)
```

Comment:
I think this also needs to check for changes and reset the error if there are any

Reply:
Makes sense